playbook-website/app/components/hp_solutions_component.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class HpSolutionsComponent < Playbook::KitBase
+  def render_code(text, language)
+    formatter = Rouge::Formatters::HTML.new(scope: ".highlight")
+    lexer = Rouge::Lexer.find(language)
+    formatter.format(lexer.lex(text))
+  end
```

Comment:
[This is already defined here](https://github.com/powerhome/playbook/blob/master/playbook-website/lib/markdown_helper.rb#L58-L62) and shouldn't be duplicated. If you cannot access it I would suggest trying to including it. Example: https://github.com/powerhome/playbook/blob/master/playbook-website/app/controllers/application_controller.rb#L6